package.json

```diff
@@ -33,18 +33,19 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@authenio/xml-encryption": "^0.11.3",
     "camelcase": "^5.3.1",
     "node-forge": "^0.8.5",
     "node-rsa": "^1.0.5",
     "pako": "^1.0.10",
     "uuid": "^3.3.2",
     "xml": "^1.0.1",
     "xml-crypto": "^1.4.0",
+    "xml-encryption": "^1.1.1",
     "xmldom": "^0.1.27",
     "xpath": "^0.0.27"
   },
   "devDependencies": {
+    "@authenio/samlify-xsd-schema-validator": "^1.0.3",
```

Comment:
No need to put in the validator here.

```suggestion
```

Reply:
Actually the tests don't pass on a fresh install locally if this devDependency isn't here. With it being a devDep, there's no risk to the user.

Reply:
Unless our users used `yarn --prod` or set `NODE_ENV=production`, it will install `devDependencies`, and I want to completely separate the core library and the importable validators.

I am also seeking for a better workaround for passing a local test, say to build a mock for the validator in order to pass the local tests. There are many validator modules that we are currently using, the test should be covered in CI.

Reply:
There is a postinstall script `javac support/XMLValidator.java` for the validator, it would also introduce unnecessary dependencies to whom develop this core library.

I fully understand the current implementation somehow makes the development side a bit unfriendly, I will do another fix to introduce **validator mock** for the test side, but i think it's better to remove those validator dependencies for this ticket. Does it make sense?

Reply:
I think mocking in tests would make tests less reliable

Reply:
@sibelius The tests with real validator will be performed and ran in Travis CI for every pull requests.

Reply:
Feel free to continue the discussion on #358. :D

Reply:
dep removed :)